Wikipedia/Code/WMFArticleFooterMenuViewController.h

```diff
@@ -0,0 +1,17 @@
+//
+//  WMFArticleFooterMenuViewController.h
+//  Wikipedia
+//
+//  Created by Monte Hurd on 12/15/15.
+//  Copyright © 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface WMFArticleFooterMenuViewController : UIViewController
+
+@property (nonatomic, strong) MWKDataStore* dataStore;
```

Comment:
classes should be immutable whenever possible: make this property `readonly` and add as a property to the initializer.